examples/rules_ios/Lib/README.md

```diff
@@ -0,0 +1,4 @@
+# CommandLineTool
+
+This is an example of an extra file that is added to the Xcode project via
```

Comment:
I need for this. In general all the "integration" stuff can get culled. A concise example will be better than the kitchen sink that Integration is trying to be.

Reply:
SGTM